src/frontend/helperfunctions/watchLaterHandler.js

```diff
@@ -51,16 +53,19 @@ export const postWatchLaterHandler = async (
 export const deleteWatchLaterHandler = async (
   token,
   videoId,
-  dataStoreDispatch
+  dataStoreDispatch,
+  skip1,
+  skip2,
```

Comment:
I think this parameters haven't used in function so whats the point of passing it ?

Reply:
Because I m passing parameters of other functions from same place, and i want toast at 6th parameter , that's why.